### ISSUE TITLE
light curve created using a TESS target pixel file with .interact fails to open

### ISSUE THREAD

**Issue by mrtommyb** [KeplerGO/lightkurve#452]
#### Example

```python
import lightkurve as lk
tpfs = lk.search_targetpixelfile('TIC 206544316').download_all()
tpf = tpfs[1]
tpf.interact('http://localhost:8889')
% Saved file tess2018234235059-s0002-0000000206544316-0121-s_tp-custom-lc.fits 
lc = lk.TessLightCurveFile('tess2018234235059-s0002-0000000206544316-0121-s_tp-custom-lc.fits', )
```
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
~/anaconda3/envs/exoplanet/lib/python3.7/site-packages/astropy/io/fits/column.py in _get_index(names, key)
   2007         try:
-> 2008             indx = names.index(key.rstrip())
   2009         except ValueError:

ValueError: 'QUALITY' is not in list

During handling of the above exception, another exception occurred:

KeyError                                  Traceback (most recent call last)
<ipython-input-14-80ea932783c4> in <module>
----> 1 lc = lk.TessLightCurveFile('tess2018234235059-s0002-0000000206544316-0121-s_tp-custom-lc.fits', )

~/anaconda3/envs/exoplanet/lib/python3.7/site-packages/lightkurve-1.0b30.dev0-py3.7.egg/lightkurve/lightcurvefile.py in __init__(self, path, quality_bitmask, **kwargs)
    382         self.quality_bitmask = quality_bitmask
    383         self.quality_mask = TessQualityFlags.create_quality_mask(
--> 384                                 quality_array=self.hdu[1].data['QUALITY'],
    385                                 bitmask=quality_bitmask)
    386         # Early TESS releases had cadences with time=NaN (i.e. missing data)

~/anaconda3/envs/exoplanet/lib/python3.7/site-packages/astropy/io/fits/fitsrec.py in __getitem__(self, key)
    485 
    486         if isinstance(key, str):
--> 487             return self.field(key)
    488 
    489         # Have to view as a recarray then back as a FITS_rec, otherwise the

~/anaconda3/envs/exoplanet/lib/python3.7/site-packages/astropy/io/fits/fitsrec.py in field(self, key)
    675         # NOTE: The *column* index may not be the same as the field index in
    676         # the recarray, if the column is a phantom column
--> 677         column = self.columns[key]
    678         name = column.name
    679         format = column.format

~/anaconda3/envs/exoplanet/lib/python3.7/site-packages/astropy/io/fits/column.py in __getitem__(self, key)
   1613     def __getitem__(self, key):
   1614         if isinstance(key, str):
-> 1615             key = _get_index(self.names, key)
   1616 
   1617         x = self.columns[key]

~/anaconda3/envs/exoplanet/lib/python3.7/site-packages/astropy/io/fits/column.py in _get_index(names, key)
   2015                 indx = names.index(_key)
   2016             elif count == 0:
-> 2017                 raise KeyError("Key '{}' does not exist.".format(key))
   2018             else:              # multiple match
   2019                 raise KeyError("Ambiguous key name '{}'.".format(key))

KeyError: "Key 'QUALITY' does not exist."
```

**Comment by mrtommyb** [comment]
ping @barentsen

**Comment by mrtommyb** [comment]
(also @barentsen , can you let me assign issues tags)

**Comment by barentsen** [comment]
@mrtommyb You now have write access to the Lightkurve repo, which includes the power to assign labels/tags!

**Comment by mrtommyb** [comment]
Sweet. I assigned it to you.

**Comment by gully** [comment]
Hi @mrtommyb  --
Remarkably `lightkurve` does not presently support the opening of these custom lightcurve objects.  This comment of an outdated issue alludes to the subtle explanation for why:
https://github.com/KeplerGO/lightkurve/issues/264#issuecomment-437201304

In short, these custom lightcurve objects are non-stardard.  They could have been produced through any number of lossy modes---like PRF photometry---and so metadata is incomplete.  **The fastest way to reuse a lightcurve produced from interact is probably the following pattern**:

```python
tpf.interact('http://localhost:8889')
% Saved file tess2018234235059-s0002-0000000206544316-0121-s_tp-custom-lc.fits 
custom_mask = fits.open('tess2018234235059-s0002-0000000206544316-0121-s_tp-custom-lc.fits')['APERTURE'].data
lc = tpf.to_lightcurve(aperture_mask=custom_mask)
```

and **voilà!**  Not only do you have an easy-to-use lightcurve object in two lines, you also have all the associated metadata you expect in a `TessLightCurve` object.  Admittedly, this programming pattern should be considered a work around until the team agrees on a *lightkurvian* way to consume these custom lightcurves.  I have recommended that we tackle this problem when/if we add support for HLSPs, since those custom lightcurves share the same problem: they're not pipeline produced, and they have disparate, additional, or incomplete metadata.

**Comment by barentsen** [comment]
This issue has been resolved by #466.
